M2-Inc/Deeb8
# Frontend Progress

WHAT DOES THIS PR DO? 
1. Adds in a new frontend component

HOW TO TEST THIS PR:
1. I added a new package, so run `npm install`. 
2. Run `npm run next-dev`. This starts the dev server on localhost:3000 
3. Interact with webpage. Login, signup, and display for groups should work, but are not integrated with backend ( will not get a response back from fetches )
